bloop.rb

```diff
@@ -48,6 +48,13 @@ def install
 
       prefix.install "bin"
   end
+  
+  service do
```

Comment:
Could you maybe add a bit more information what does this do and why it is needed? Thanks!